lambda_function.py

```diff
@@ -55,6 +68,20 @@ def handler(event, context):
         else:
             response_status = cfnresponse.FAILED
             cfnresponse.send(event, context, response_status, response_data)
+    
+    elif event["ResourceType"] == "Custom::R53HostedZone":
+        log.debug("R53 Hosted Zone lookup")
+        r53 = boto3.client("route53")
+        try: zone_id = event["ResourceProperties"]["HostedZoneId"]
```

Comment:
Same formatting error here

Reply:
OK I think I have an idea how to do this, so will just merge it as it is and, then change it ( as agreed )